lib/carto/url_validator.rb

```diff
@@ -22,6 +22,7 @@ def valid_url?(str, valid_ports)
       uri = URI.parse(str)
 
       (uri.is_a?(URI::HTTP) || uri.is_a?(URI::FTP)) &&
+        valid_ports.include?(uri.port) &&
```

Comment:
What about adding Riak ports (and other services you guys use) into `DEFAULT_URL_VALID_PORTS `

Reply:
That may not be necessary.  We still use the standard ports when making the requests from editor.  However nginx will then forward it to the correct port behind the scenes.